backend/ee/danswer/server/query_history/api.py

```diff
@@ -330,6 +333,36 @@ def snapshot_from_chat_session(
     )
 
 
+@router.get("/admin/chat-sessions")
```

Comment:
nit: seems worth defining a new admin_router for the admin endpoints in query history api